certbot/certbot/_internal/tests/storage_test.py

```diff
@@ -480,6 +492,27 @@ def test_time_interval_judgments(self, mock_datetime, mock_set_by_user):
                 (1420070400, "10 minutes", True),
                 (1420070400, "10 weeks", True), (1420070400, "10 months", True),
                 (1420070400, "10 years", True), (1420070400, "99 months", True),
+                (1420070400, None, True)
+        ]:
+            sometime = datetime.datetime.fromtimestamp(current_time, pytz.UTC)
+            mock_datetime.datetime.now.return_value = sometime
+            self.test_rc.configuration["renew_before_expiry"] = interval
+            assert self.test_rc.should_autorenew() == result
+
+        # Lifetime: 31 years
```

Comment:
:scream:

Reply:
I actually wound up replacing this with having the test generate its own certs since it depends so specifically on their lifetimes. And there was no particular reason that cert_2048 should have such a long life, so in #10213 it doesn't.